### ISSUE TITLE
LCMC 1.7.10 no longer works with RSA/DSA keys

### ISSUE THREAD

**Issue by sammcj** [rasto/lcmc#59]
since upgrading to 1.7.10 there is no prompt to enter your password in the wizard or on your existing connections thus, there is no way to hit enter to make it just use your ssh keys, which always used to work.

* OSX 10.11.4
* Java(TM) SE Runtime Environment (build 1.8.0_51-b16)
* Java HotSpot(TM) 64-Bit Server VM (build 25.51-b03, mixed mode)

![screen shot 2016-03-30 at 3 07 24 pm](https://cloud.githubusercontent.com/assets/862951/14131697/7f0e6d8e-f689-11e5-93b5-4d06cfdf0729.jpg)

**Comment by sammcj** [comment]
This is still a problem with 1.7.11

![screen shot 2016-05-23 at 11 08 26 am](https://cloud.githubusercontent.com/assets/862951/15457946/d15ed368-20d6-11e6-975c-847afbc87289.jpg)

**Comment by sammcj** [comment]
On the host:

```
root@s1-san6: # lcmc-gui-helper-1.7.8
Died at /usr/local/bin/lcmc-gui-helper-1.7.8 line 51.
```

Looking at line 51 of that file:

```
my $action = shift @$ACTION_OPTIONS || die;
```

**Comment by sammcj** [comment]
Think I found the problem after enabling auditd on the server side:

```bash
type=CRYPTO_SESSION msg=audit(1463966687.274:502): pid=9614 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:sshd_t:s0-s0:c0.c1023 msg='op=unsupported-mac direction=? cipher=? ksize=? rport=63317 laddr=10.51.40.61 lport=22  exe="/usr/sbin/sshd" hostname=? addr=10.51.15.92 terminal=? res=failed'
```

It looks like LCMC is trying to connect with some sort of unsupported cipher - perhaps a deprecated, old cypher?

Cyphers supported by OpenSSH on CentOS7 using recommended settings:

```
KexAlgorithms curve25519-sha256@libssh.org,ecdh-sha2-nistp521,ecdh-sha2-nistp384,ecdh-sha2-nistp256,diffie-hellman-group-exchange-sha256

Ciphers chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes192-ctr,aes128-ctr

MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,umac-128@openssh.com
```

**Comment by sammcj** [comment]
@rasto - any input on this?

**Comment by przemas75** [comment]
Yes, it uses the old version of https://github.com/maxd/ganymed-ssh-2 for ssh connections.
Here: https://github.com/rasto/lcmc/blob/master/src/lib/ch/ethz/ssh2/crypto/digest/MAC.java
One can see only these MACs in use:
`hmac-sha1-96", "hmac-sha1", "hmac-md5-96", "hmac-md5" `
Very poor.
I have tried to update, but there is a change in scp in Ganymed, and further steps to adjust lcmc.

**Comment by danyel2014** [comment]
The issue is still present on 1.7.14 . Are there any plans to fix this ?

Thank you
